src/builders/JsonapiRequestBuilder.js

```diff
@@ -0,0 +1,167 @@
+import { merge } from 'lodash';
+import JsonapiResourceReader from './JsonapiResourceReader';
+import JsonapiResourceWriter from './JsonapiResourceWriter';
+
+import { requestActionType, mergeParamsInUrlPlaceholdersAndParams } from '../utils/builders';
+
+export default class JsonapiRequestBuilder {
+  /**
+   * @param  {Object} params
+   * @param  {JsonapiResourceBuilder} params.resource
+   * @param  {String} params.httpMethod
+   * @param  {String} params.path Path of the request
```

Comment:
`endpoint` instead of `path` ?

Reply:
Oui parce que justement path c'est le path sans le host alors que l'endpoint ce serait le host quoi. L'endpoint provient de l'API, le path étant le "chemin relatif" de la requête